evidence/reactor.go

```diff
@@ -212,52 +206,54 @@ func (evR Reactor) checkSendEvidenceMessage(
 	}
 
 	// send evidence
-	msg = &ListMessage{[]types.Evidence{ev}}
-	return msg, false
+	return []types.Evidence{ev}, false
 }
 
 // PeerState describes the state of a peer.
 type PeerState interface {
 	GetHeight() int64
 }
 
-//-----------------------------------------------------------------------------
-// Messages
-
-// Message is a message sent or received by the Reactor.
-type Message interface {
-	ValidateBasic() error
-}
+// encodemsg takes a array of evidence
+// returns the byte encoding of the List Message
+func encodeMsg(evis []types.Evidence) ([]byte, error) {
+	evi := make([]*tmproto.Evidence, len(evis))
+	for i := 0; i < len(evis); i++ {
+		le := evis[i]
```

Comment:
not sure why we need this interim variable.. it does not seem to add any meaning